cql3/statements/create_keyspace_statement.cc

```diff
@@ -208,7 +208,7 @@ create_keyspace_statement::execute(query_processor& qp, service::query_state& st
     });
 }
 
-lw_shared_ptr<keyspace_metadata> create_keyspace_statement::get_keyspace_metadata(const locator::token_metadata& tm) {
+lw_shared_ptr<data_dictionary::keyspace_metadata> create_keyspace_statement::get_keyspace_metadata(const locator::token_metadata& tm) {
```

Comment:
I can't place a comment on the commit message, so I'll do so here. Typo: s/moce/move